optimum/onnxruntime/utils.py

```diff
@@ -127,6 +127,7 @@ class ORTConfigManager:
         "nystromformer": "bert",
         "pegasus": "bert",
         "roberta": "bert",
+        "stablelm": "gpt2",
```

Comment:
Not sure if it's gpt2-like.

Reply:
I think it is correct, [this comment](https://github.com/huggingface/transformers/blob/main/src/transformers/models/stablelm/modeling_stablelm.py#L4) mentions it's similar to gpt-neox with small mods.

Reply:
Thank you.